modules/exploits/osx/local/persistence.rb

```diff
@@ -47,7 +54,8 @@ def initialize(info = {})
       ),
       OptBool.new('RUN_NOW',
         [false, 'Run the installed payload immediately.', false]
-      )
+      ),
+      OptEnum.new('LAUNCH_ITEM', [false, 'Type of launch item, see description for more info. Default is LaunchAgent', 'LaunchAgent', %w[LaunchAgent LaunchDaemon]])
```

Comment:
```suggestion
      OptEnum.new('LAUNCH_ITEM', [true, 'Type of launch item, see description for more info. Default is LaunchAgent', 'LaunchAgent', %w[LaunchAgent LaunchDaemon]])
```
seems like this is required